community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/legacystore/v19/Legacy19Store.java

```diff
@@ -45,18 +51,36 @@
  * <p>
  * {@link #LEGACY_VERSION} marks which version it's able to read.
  */
-public class Legacy19Store implements LegacyStore
+public class Legacy19Store implements LegacyStore, LegacyTokenStoreHighIdReader
 {
     public static final String LEGACY_VERSION = "v0.A.0";
 
+    // always one for token stores
+    private static final int bytesRequiredToDetermineInUse = 1;
```

Comment:
Can we please name these constants capital case with `_`?
Same in `Legacy20Store`